lib/samson/tasks/lock_cleaner.rb

```diff
@@ -0,0 +1,26 @@
+module Samson::Tasks
+  class LockCleaner
+
+    def self.start
+      new.start
+    end
+
+    def start
+      task.tap(&:execute)
+    end
+
+    def task
+      @task ||= Concurrent::TimerTask.new(run_now: true, execution_interval: 60, timeout_interval: 10) do
+        Lock.remove_expired_locks
+      end.with_observer(self)
```

Comment:
Yeah, my bad, the docs say "instantiation" but I should've tried it
https://github.com/ruby-concurrency/concurrent-ruby/blob/master/lib/concurrent/timer_task.rb#L171-L173